Unit test failure after last merges.

```
>       assert f'"type": "{NodeDriverTestException.__name__}"' in j
E       assert '"type": "NodeDriverTestException"' in '{\n    "plan": {\n        "sessions": [\n            {\n                "constellation": {\n                    "role...name": null,\n    "hostname": null,\n    "type": "feditest-testrun-transcript",\n    "feditest_version": "feditest"\n}'

tests/test_report_node_driver_errors.py:92: AssertionError
----------------------------- Captured stderr call -----------------------------
ERROR    feditest:reporting.py:108 Ended TestRunSession Unnamed with Exception:
Traceback (most recent call last):
  File ".../feditest/src/feditest/testrun.py", line 342, in run
    self.run_constellation.setup()
  File ".../feditest/src/feditest/testrun.py", line 84, in setup
    node : Node = node_driver.provision_node(plan_role_name, plan_node)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../feditest/src/feditest/protocols/__init__.py", line 131, in provision_node
    ret = self._provision_node(rolename, test_plan_node, parameters)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: init.<locals>.Faulty_NodeDriver._provision_node() takes 3 positional arguments but 4 were given
```

--- a/tests/test_report_node_driver_errors.py
+++ b/tests/test_report_node_driver_errors.py
@@ -51,7 +51,12 @@ def init():
 
     @nodedriver
     class Faulty_NodeDriver(NodeDriver):
-        def _provision_node(self, rolename: str, parameters: dict[str,Any]) -> Node:
+        def _provision_node(
+            self,
+            olename: str,
+            test_plan_node: TestPlanConstellationNode,
+            parameters: dict[str, Any],
+        ) -> Node:
             raise NodeDriverTestException()
 
     feditest._loading_node_drivers = False

